doc/admin-guide/files/records.config.en.rst

```diff
@@ -2422,6 +2422,20 @@ DNS
    in DNS Injection attacks), particularly in forward or transparent proxies, but
    requires that the resolver populates the queries section of the response properly.
 
+.. ts:cv:: CONFIG proxy.config.dns.connection.mode INT 0
+
+   Three connection modes between |TS| and nameservers can be set -- UDP_ONLY,
+   TCP_RETRY, TCP_ONLY.
+
+
+   ===== ======================================================================
+   Value Description
+   ===== ======================================================================
+   ``0`` UDP_ONLY:  |TS| always talks to nameservers over UDP.
+   ``1`` TCP_RETRY: |TS| will retry queries over TCP when truncated flag is set.
```

Comment:
This is slightly confusing. What does "truncated flag" mean (I assume it means the query is too long to fit a UDP message)? It looks to me from open_cons() that if set to "1", it will create both the UDP and the TCP connection, always? If so, where does the size come in there? Regardless, maybe a better description here, such as "Try UDP first, but fall back to TCP if query is too large" ?